src/go/k8s/tests/e2e/produce-tls/02-create-config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
merge this with the next step

Reply:
on second thought, this has to be separate so we know this is created before we use it in the job

Reply:
On of the Kubernetes controller should retry if config map doesn't exist. After few milliseconds it should start a pod as whole distributed pieces are in sync :)

Reply:
yeah right... but in the end, is it such a big deal to have this separate? This way I can ensure the order :)

Reply:
It can stay as is.